From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Tue, 8 Feb 2022 21:09:20 +0100
Subject: perl: don't build in parallel and bump release

Parallel builds cause spurious build failures with high core counts.

https://github.com/openwrt/packages/issues/8238
https://github.com/openwrt/packages/pull/17274

diff --git a/lang/perl/Makefile b/lang/perl/Makefile
index 6b748019bc49c7952948b35811b206bfeacfaa66..75c08151b395d4671da755ffbfa77ef1b40e36df 100644
--- a/lang/perl/Makefile
+++ b/lang/perl/Makefile
@@ -27,8 +27,8 @@ PKG_BUILD_DIR:=$(BUILD_DIR)/perl/$(PKG_NAME)-$(PKG_VERSION)
 HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/perl/$(PKG_NAME)-$(PKG_VERSION)
 PKG_INSTALL:=1
 PKG_BUILD_DEPENDS:=perl/host
-PKG_BUILD_PARALLEL:=1
-HOST_BUILD_PARALLEL:=1
+PKG_BUILD_PARALLEL:=0
+HOST_BUILD_PARALLEL:=0
 
 # Variables used during configuration/build
 HOST_PERL_PREFIX:=$(STAGING_DIR_HOSTPKG)/usr
